### ISSUE TITLE
Create the footer

### ISSUE THREAD

**Issue by ifr4h** [chingu-voyages/v37-toucans-team-01#20]
**__Task Description__**
This will be seen on all pages within the website

**__Technical Considerations__**
This should be responsive and accessibility should be considered when creating it 

**__Additional Considerations__**
Any supplemental information including unresolved questions, links to external resources, screenshots, etc.

**Comment by ifr4h** [comment]
The footer should stay at the bottom of the page and not stick to the bottom of the screen as it covers content
